### ISSUE TITLE
Android background streaming crash: Bad notification for startForeground

### ISSUE THREAD

**Issue by chungweileong94** [red5pro/react-native-red5pro#59]
On Android (RN 0.63.1), if the app go to background while stream (with background streaming enable), the app will crash.

Here's the error: 
```
android.app.RemoteServiceException: Bad notification for startForeground: java.lang.RuntimeException: invalid channel for service notification: Notification(channel=null pri=0 contentView=null vibrate=null sound=null defaults=0x0 flags=0x40 color=0x00000000 vis=PRIVATE)
```

**Comment by chungweileong94** [comment]
I submitted a PR https://github.com/red5pro/react-native-red5pro/pull/60 for the fix.

**Comment by chungweileong94** [comment]
Resolved in #60
